views/index.html

```diff
@@ -19,24 +19,28 @@ <h1 class="text-center">Tic Tac Toe</h1>
 
   <div id = "game-view" class='container board text-center'>
     <div id = "game-view-info">
-      <div class="scores"></div>
-        <div class="result"></div>
-        <button class="btn btn-dark" id = "restart">Restart Game</button>
+      <div class="row">
+          <div class="col-lg-4"><div class="scores"></div></div>
+          <div class="col-lg-4"><div class="result"></div></div>
+          <div class="col-lg-4"><button class="btn btn-transparent" id = "restart">Restart</button></div>
+          
+      </div>
+     
   </div>
   
     
-<div class="game-view-sqaures">
-    <div class="row">
+<div id="game-view-sqaures">
```

Comment:
instead of justifying center, make this flexbox
```
    display: flex;
    flex-direction: column;
    flex-wrap: nowrap;
    align-items: center;
```
Make sure you use bootstap classes for flex